vxyowsup/whatsapp.py

```diff
@@ -154,8 +154,8 @@ def onMessage(self, messageProtocolEntity):
             messageProtocolEntity.getParticipant())
         self.toLower(receipt)
 
-        print "You have received a message, and thusly sent a receipt"
-        print "You are now sending a reply"
+        log.debug('You have received a message, and thusly sent a receipt')
+        # log.debug("You are now sending a reply")
         # self.send_to_human('iiii', from_address + '@s.whatsapp.net', "this transport's gone rogue")
```

Comment:
That might be helpful.. But surely this would be equivalent to sending it to just one MSISDN?  (currently mine) Or is this for debugging when it's already sending to actual users?